gallery/gallery/lib/demos/cupertino/cupertino_tab_bar_demo.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN cupertinoNavigationDemo
+
+class _TabInfo {
+  const _TabInfo(this.title, this.icon);
+
+  final String title;
+  final IconData icon;
+}
+
+class CupertinoTabBarDemo extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final _tabInfo = <_TabInfo>[
```

Comment:
The `<TabInfo>` here can be dropped. Dart should infer that type correctly.

Reply:
Done